website/search/elastic_search.py

```diff
@@ -267,7 +267,7 @@ def format_result(result, parent_id=None):
         'is_pending_retraction': result['is_pending_retraction'],
         'embargo_end_date': result['embargo_end_date'],
         'is_pending_embargo': result['is_pending_embargo'],
-        'description': result['description'],
+        'description': sanitize.unescape_entities(result['description']),
```

Comment:
Interesting, the best place for the sanitization I need to do on the component description is in python (directly injected via mako)-- is this a safe way to do that if ended up in html instead of elastic? No worries if unsure, I'll dig deeper tomorrow.